src/commands/miscellaneous/ping.ts

```diff
@@ -1,42 +1,22 @@
 import { Command, container } from '@sapphire/framework';
-import { Message } from 'discord.js';
 import {
   CodeyCommand,
   CodeyCommandDetails,
-  CodeyCommandResponseType,
   SapphireMessageExecuteType,
-  SapphireMessageRequest,
   SapphireMessageResponse
 } from '../../codeyCommand';
 
-const initialPingContent = 'Ping?';
-
-const getApiLatency = (initialPing: SapphireMessageRequest, messageFromUserAsMessage: Message<boolean>) => {
-  const initialPingAsMessage = <Message<boolean>>initialPing;
-  return initialPingAsMessage.createdTimestamp - messageFromUserAsMessage.createdTimestamp;
-};
-
 const content = (botLatency: number, apiLatency: number) =>
   `Pong from JavaScript! Bot Latency ${botLatency}ms. API Latency ${apiLatency}ms.`;
 
 const executeCommand: SapphireMessageExecuteType = async (
   client,
   messageFromUser,
-  _args,
-  initialMessageFromBot
+  _args
 ): Promise<SapphireMessageResponse> => {
-  // Assert message types are Message<boolean>
-  // We have to do this because APIMessage does not have "createdTimestamp" property
-  const messageFromUserAsMessage = <Message<boolean>>messageFromUser;
   const botLatency = client.ws.ping;
-  if (initialMessageFromBot) {
-    const apiLatency = getApiLatency(initialMessageFromBot, messageFromUserAsMessage);
-    return new Promise((resolve, _reject) => resolve(content(botLatency, apiLatency)));
-  }
-  const initialPing = await messageFromUserAsMessage.channel.send(initialPingContent);
-  const apiLatency = getApiLatency(initialPing, messageFromUserAsMessage);
-  initialPing.edit(content(botLatency, apiLatency));
-  return Promise.resolve('');
+  const apiLatency = Date.now() - messageFromUser.createdTimestamp;
+  return `${content(botLatency, apiLatency)}`;
 };
```

Comment:
The sole exception for an edited message should be the `ping` command (i.e. even the regular command edits the message for `ping`). The previous implementation makes sense since it is "precisely" the time it takes for the bot to respond, which is what we want to measure.

Reply:
Fair point! I'll rework the impl

Reply:
Done in [60c2b16](https://github.com/uwcsc/codeybot/pull/329/commits/60c2b16f86a7de34bd06d70fa202223a0aedf3b5)